### PR TITLE
Fix tokyotosho failing test and add type

### PR DESCRIPTION
**Pull request by Chao-Man**
- Fixed definition failing test, https://www.tokyotosho.info/ redirects to https://www.tokyotosho.info/index.php
- Changed language to UTF-8.
- Added "public" type

### CODE REVIEW

File: src/Jackett/Definitions/tokyotosho.yml [hunk 1]
```diff
@@ -2,7 +2,8 @@
   site: tokyotosho
   name: Tokyo Toshokan
   description: "A BitTorrent Library for Japanese Media"
-  language: en-us
```
Comment (kaso17) [inline src/Jackett/Definitions/tokyotosho.yml]: language is fine

File: src/Jackett/Definitions/tokyotosho.yml [hunk 2]
```diff
@@ -2,7 +2,8 @@
...
+  language: UTF-8
```
Comment (kaso17) [inline src/Jackett/Definitions/tokyotosho.yml]: this should be encoding (not language)

**Comment by Chao-Man** [comment]
That's my bad, thanks for the feedback.
I've reverted the change.

**Comment by TGM** [comment]
" Test failed for tokyotosho: Found no results while trying to browse this tracker"
Are you sure it's working properly?

**Comment by Chao-Man** [comment]
@TGM I'm pretty sure it's working properly.

screen shot 2017-05-11 at 10 33 08 am
screen shot 2017-05-11 at 10 33 19 am

**Comment by PnoT** [comment]
I've tried the test and the manual search, as you posted, and still nothing is working.  A few of us are posting here https://github.com/Jackett/Jackett/issues/1307

**Comment by Chao-Man** [comment]
Unfortunately I can't reproduce the test fail condition. Can anyone produce logs when using my fork?

Or use the following config:
https://raw.githubusercontent.com/Chao-Man/Jackett/master/src/Jackett/Definitions/tokyotosho.yml

My logs: https://pastebin.com/g8r9C1kY

Mono Stable 4.8.1.0/22a39d7
Ubuntu Server 16.04 LTS

**Pull request merged**